contrib/kind-common.sh

```
@@ -1676,3 +1722,156 @@ label_ovn_ha() {
+  KIND_NODES=$(kind get nodes --name "${KIND_CLUSTER_NAME}")
+  for n in $KIND_NODES; do
+    docker exec "$n" ip route delete default
+  done
+}
+
+add_dns_hostnames() {
+  dns="$'"
+  KIND_NODES=$(kind get nodes --name "${KIND_CLUSTER_NAME}")
+  # find all IPs and build dns entries
+  for n in $KIND_NODES; do
+	ip=$(docker container inspect -f '{{ .NetworkSettings.Networks.kind.IPAddress }}' $n)
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)